cmd/query/app/server.go

```diff
@@ -105,10 +104,11 @@ func (s *Server) Start() error {
 	if port, err := netutils.GetPort(s.conn.Addr()); err == nil {
 		tcpPort = port
 	}
+	s.svc.Logger.Info("addr", zap.Int("port", tcpPort))
 
 	s.svc.Logger.Info(
 		"Query server started",
-		zap.Int("port", tcpPort))
+		zap.String("addr", s.queryOptions.HostPort))
```

Comment:
keep the port field as well, because starting with `:0` is valid, and the `tcpPort` var gives the actual port number.

same for the next 3 logs

Reply:
refactored as suggested